Vendor/iOS/MSAlertController/MSAlertController.m

```diff
@@ -146,4 +146,30 @@ + (void)presentNextAlertAnimated:(BOOL)animated {
   }
 }
 
+#define MS_DISPATCH_SELECTOR(result, class, selectorName, ...) ({ \
```

Comment:
Can you move this to the Utility class? I could see this in another PR, let's avoid having duplicated code here and there.

Reply:
No, it's in a `Vendor` folder

Reply:
Right, but can you move it up in this file so it's not in between two instance or class methods :) ?